pages/_app.jsx

```diff
@@ -1,7 +1,50 @@
+import { useEffect, useState } from "react";
 import "@/styles/globals.scss";
 import Header from "@/components/Header";
 
 export default function App({ Component, pageProps }) {
+	const [theme, setTheme] = useState("bright");
```

Comment:
The idea of the state residing in the app.jsx is correct but it should be inside a custom context to enable the global access that prevent prop drilling.